slack_sdk/webhook/async_client.py

```diff
@@ -19,7 +19,15 @@
 
 
 class AsyncWebhookClient:
-    logger = logging.getLogger(__name__)
+    url: str
```

Comment:
These additions are intended to improve developer experience (having these type hints improves compatibility with editors like PyCharm and code analyzers like pytype)